### ISSUE TITLE
[MMP] Reply counts are occasionally the same seemingly random large number for all threads

### ISSUE THREAD

**Issue by TheEnigmaBlade** [creesch/reddit-moderator-toolbox#420]
Reply counts are fine in debug output, but the displayed value is a random large number.

Example:

![](http://i.imgur.com/g9Hpvsk.png)

**Comment by TheEnigmaBlade** [comment]
Seemingly fixed with 413370a13fd975fca5a53132836a767a9e18c944, but will keep an eye out for it happening again.

**Comment by TheEnigmaBlade** [comment]
Confirmed fixed for good. Found the root cause (after fixing) from reddit's javascript:

```javascript
function unread_thing(e) {
    var t = $(e);
    t.hasClass("thing") || (t = t.thing()),
    alterUnreadCount(1),
    $(t).addClass("new unread")
}
 
function alterUnreadCount(e) {
    var t = $(".message-count");
    t.length > 0 && t.text(parseInt(t.text(), 10) + e)
} 
```
